lpa-submissions-e2e-tests/cypress/integration/appealsPlanningOfficersReport.feature

```diff
@@ -0,0 +1,97 @@
+Given LPA Planning Officer is within the LPA Questionnaire task list
+When LPA Planning Officer selects the Upload the Planning Officers Report link
+Then LPA Planning Officer is presented with the ability to upload the Planning Officers report
+
+Scenario Outline: AC2 LPA Planning Officer successfully uploads file via upload button
+Given Upload the Planning Officers report is requested
+When a '<valid_file>' is successfully uploaded
+Then progress is made to the Task List
+And Upload the Planning Officers report is shown as completed
+Examples:
+|valid_file|
+|upload-file-valid.tiff|
+|upload-file-valid.tif|
+|upload-file-valid.png|
+|upload-file-valid.pdf|
+|upload-file-valid.jpg|
+|upload-file-valid.jpeg|
+|upload-file-valid.doc|
+|upload-file-valid.docx|
+
+Scenario Outline: AC3 LPA Planning Officer successfully uploads file via Drag and Drop
+Given Upload the Planning Officers report is requested
+When a '<valid_file>' is uploaded via drag and drop
+Then progress is made to the Task List
+And Upload the Planning Officers report is requested
+Examples:
+|valid_file|
+|upload-file-valid.tiff|
+|upload-file-valid.tif|
+|upload-file-valid.png|
+|upload-file-valid.pdf|
+|upload-file-valid.jpg|
+|upload-file-valid.jpeg|
+|upload-file-valid.doc|
+|upload-file-valid.docx|
+
+Scenario Outline: AC4 LPA Planning Officer successfully uploads multiple files
+Given Upload the Planning Officers report is requested
+When valid '<multiple_files>' are uploaded
+Then progress is made to the Task List
+And Upload the Planning Officers report is shown as completed
+Examples:
+|multiple_files|
+|upload-file-valid.tiff, upload-file-valid.tif, upload-file-valid.png |
+|upload-file-valid.pdf, upload-file-valid.jpg, upload-file-valid.jpeg |
+|upload-file-valid.doc, upload-file-valid.docx |
+
+Scenario: AC5 LPA Planning Officer does not upload a file and is provided with an error
+Given Upload the Planning Officers report is requested
+When no file has been uploaded
+Then progress is halted with error message 'Upload the planning officer’s report or other documents and minutes'
+
+Scenario Outline: AC6 LPA Planning Officer selects invalid file size
+Given Upload the Planning Officers report is requested
+When files of '<invalid_file_size>' have been selected
+Then progress is halted with error message 'The file must be smaller than 15MB'
+Examples:
+|invalid_file_size|
+|upload_file_large|
+
+Scenario Outline: AC7 LPA Planning Officer selects Invalid file format
+Given Upload the Planning Officers report is requested
+When files of '<invalid_format>' have been selected
+Then progress is halted with error message 'The file must be DOC, DOCX, PDF, TIF, JPG or PNG'
+Examples:
+|invalid_format|
+|upload-file-invalid-wrong-type.csv|
+
+Scenario: AC8 LPA Planning Officer selects to return to previous page
+Given Upload the Planning Officers report is requested
+When Back is then requested
+Then the Task list is presented
```

Comment:
Do we need to add this: And any document upload will not be saved

Reply:
The equivalent wasn't included in AS-9 so I took my cue from there. @shilpabasu22 ?

Reply:
it wasn't in original acceptance criteria, if you add these @pippalou , we can update the tests